VTEX - Orders API (PII version).json

```diff
@@ -3569,49 +3570,156 @@
                 "operationId": "ListOrders2",
                 "parameters": [
                     {
                         "$ref": "#/components/parameters/Content-Type"
                     },
                     {
                         "$ref": "#/components/parameters/Accept"
-                    },
-                    {
-                        "$ref": "#/components/parameters/f_hasInputInvoice"
                     }
                 ],
                 "requestBody": {
                     "content": {
                         "application/json": {
                             "schema": {
                                 "type": "object",
-                                "required": [
-                                    "f_creationDate",
-                                    "per_page",
-                                    "page"
-                                ],
                                 "properties": {
+                                    "f_status": {
+                                        "type": "string",
+                                        "description": "Filters orders by [order status](https://help.vtex.com/en/tutorial/order-flow-and-status--tutorials_196). The possible values are:\r\n- `waiting-for-sellers-confirmation`\r\n- `payment-pending`\r\n- `payment-approved`\r\n- `ready-for-handling`\r\n- `handling`\r\n- `invoiced`\r\n- `canceled`",
+                                        "example": "ready-for-handling"
+                                    },
                                     "f_creationDate": {
                                         "type": "string",
-                                        "description": "Concatened value sufix {{creationDate}} and range date in Timestamp format.",
-                                        "example": "creationDate:[2021-11-01T00:00:00.000Z TO 2022-11-10T02:00:00.000Z]"
+                                        "description": "Filters orders by creation date by concatenating the sufix `creationDate` with the desired date range in [ISO 8601 format](https://learn.microsoft.com/en-us/rest/api/storageservices/formatting-datetime-values), as in `creationDate:[YYYY-MM-DDThh:mm:ssZ TO YYYY-MM-DDThh:mm:ssZ]`. For example:\r\n- 1 day: `creationDate:[2024-01-01T00:00:00.000Z TO 2024-01-02T00:00:00.000Z]`\r\n- 1 month: `creationDate:[2024-01-01T00:00:00.000Z TO 2024-02-01T00:00:00.000Z]`\r\n- 1 year: `creationDate:[2023-01-01T00:00:00.000Z TO 2024-01-01T00:00:00.000Z]`",
+                                        "example": "creationDate:[2024-07-01T00:00:00.000Z TO 2024-08-01T00:00:00.000Z]"
+                                    },
+                                    "incompleteOrders": {
+                                        "type": "boolean",
+                                        "description": "Filters [incomplete orders](https://help.vtex.com/en/tutorial/understanding-incomplete-orders) when set as `true`, and complete orders when set as `false`.",
```

Comment:
If true, only incomplete orders will show up on the list or all orders (complete + incomplete)? I suggest explaining this here.